gtp2ogs.js

```diff
@@ -305,6 +344,16 @@ class Game {
             this.state = gamedata;
             check_for_move();
         });
+        this.socket.on('game/' + game_id + '/clock', (clock) => {
```

Comment:
My bad, this is fine here, I think I didn't click the code expandy button enough and thought this was living within the loadState call, which it most certainly is not. Code read fail.